react-hyf/react-hyf-client/src/components/Homework3.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PhotoContainer from './PhotoContainer';
+import NameHeader from './NameHeader';
+import List from './List';
+class Homework3 extends React.Component {
+  render() {
+    const { person } = this.props;
+    return (
+      <div className="homework3">
+        <NameHeader {...person}></NameHeader>
```

Comment:
Better to explicitly say which properties you are sending to the component, this way the code can become easily a bit unclear.